frontend/src/components/ForceScales/ForceScales.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import SliderControl from '../SliderControl/SliderControl.jsx';
+import styles from "./ForceScales.css";
+
+let that;
+
+/*
+    <SliderControl min='0' max='100' key={'thrust0'} indx={0} val={this.state.scales[0]} inv={this.state.inv[0]} rend={this.rendData.bind(this)} name={"Thruster 0"} />
+*/
+
+export default class ForceScale extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {'scales': props.scales};
+
+        this.rendLeftScales = this.rendLeftScales.bind(this);
+        this.rendRightScales = this.rendRightScales.bind(this);
+
+        that = this;
+    }
+
+    rendData(val, inv, key) {
+        //console.log(this.state.scales[key]);
+        let scalescpy = this.state.scales;
+        scalescpy[key] = val;
+        //scalescpy[key].invert = inv;
+        this.setState({
+            scales: scalescpy
+        }, function() {
+            this.props.rend(this.state.scales);
+        });
+    }
+
+    rendLeftScales() {
+        return Object.keys(this.props.scales).map(function(val, index) {
+            if(val.slice(0,3) === "vel")
```

Comment:
this seems like a weird way to check partition the rendering. But if you do, `val.startsWith("vel")` is a better way to check